pkg/ebpf/netlink/conntracker.go

```diff
@@ -100,6 +109,29 @@ func NewConntracker(procRoot string, stbSize int) (Conntracker, error) {
 	return ctr, nil
 }
 
+func (ctr *realConntracker) expvarStats() {
+	ticker := time.NewTicker(5 * time.Second)
+	// starts running the body immediately instead waiting for the first tick
+	for ; true; <-ticker.C {
+		stats := ctr.GetStats()
+
+		for metric, val := range stats {
+			switch v := val.(type) {
+			case int64:
+				currVal := &expvar.Int{}
+				currVal.Set(v)
+				conntrackExpvar.Set(metric, currVal)
+			case float64:
+				currVal := &expvar.Float{}
+				currVal.Set(v)
+				conntrackExpvar.Set(metric, currVal)
+			default:
+				log.Errorf("unknown value type for stats: %s, value: %v", metric, v)
```

Comment:
can we force the stats to be either `int64` or `float64` to avoid having to do casting and error handling ?

Reply:
I converted the float64 ones to int64 so we have one less case to handle, but I don't want to get rid of the default error handling just in case some one(carelessly) add another type and ended up crashing the program?

Reply:
Can we force the stats map to be `map[string]int64` to avoid that ?